gobblin-config-management/gobblin-config-core/src/main/java/gobblin/config/store/api/ConfigStoreCreationException.java

```diff
@@ -0,0 +1,17 @@
+package gobblin.config.store.api;
+
+public class ConfigStoreCreationException extends Exception {
```

Comment:
To make this exception class more useful, I'd add a storeURI attribute with a getter and also automatically include the store URI in the message.